proto/meta.proto

```diff
@@ -604,6 +604,8 @@ message SystemParams {
   optional bool enable_tracing = 15;
   optional bool use_new_object_prefix_strategy = 16;
   optional string license_key = 17;
+  optional uint32 tracing_threshold_tiered_cache_read_ms = 18;
+  optional uint32 tracing_threshold_tiered_cache_write_ms = 19;
```

Comment:
Personally I would suggest making them configuration entries (like under `[storage.cache.developer]`) instead of system parameters, since it looks to be less user-facing and does not require frequent online changes. 😕

Reply:
> since it looks to be less user-facing and does not require frequent online changes

+1. But the thing is, tail-based tracing is supposed to be enabled for only a few seconds to capture abnormal spans in the current state. Using a config file requires a restart, which may destroy the scene.

Reply:
Can we simply use default values that are small enough and rely only on `enable_tracing` to toggle it?

Reply:
Maybe enable/disable with RPC is better?

Reply:
Can you elaborate more?

Reply:
Like we do now to capture await tree and other profiling data, add a RPC function to control the tracing configurations.

https://github.com/risingwavelabs/risingwave/blob/3c102d50a5ea39ee02b9250edfd7ece6f896fc02/proto/monitor_service.proto#L68-L75